docs/tutorials/python/ndarray.md

```diff
@@ -94,9 +96,9 @@ If two `NDArray`s are located on different devices, we need to explicitly move t
 
 ## Load and Save
 
-There are two ways to save data to (or load it from) disks easily. The first way uses
-`pickle`.  `NDArray` is pickle compatible, which means that you can simply pickle the
-`NDArray` as you do with `numpy.ndarray`:
+There are two ways to save data to (or load it from) disks easily. The first way
+uses `pickle`.  A `NDArray` is pickle compatible, which means that you can
```

Comment:
"a" -> "an" because `NDArray` starts with a *vowel sound* (even though n is a consonant)